composer/core/state.py

```diff
@@ -130,6 +91,20 @@ class State(Serializable):
     loss: types.Tensors
     outputs: types.Tensors
 
+    # These attributes will be serialized using .state_dict(), and loaded with .load_state_dict()
+    # All other attributes will not be serialized.
+    # For simplicity, the leading underscore for private attributes with public getters/setters
```

Comment:
Which underscore is this referring to? Shouldn't `serialized_attributes` be `_serialized_attributes` because it is private?

Reply:
I decided to make this public since the trainer modifies it to remove the model and optimizer from the serialized fields when using deepspeed.

Reply:
Done